frontend/src/slices/auth/libs/types/auth-user.ts

```diff
@@ -0,0 +1,10 @@
+import { type UserEntityT, type UserGroupEntityT } from '~/libs/types/types.js';
+
+type AuthUserT = Pick<
```

Comment:
@PavloSukhinin said that we don't need to create a separate type for the user on the frontend. What is wrong with shared types that were in `auth.slice`? In addition, `AuthUserT` does not have a business field that will be returned for users of this category.